src/main/yaml/migration.yml

```diff
@@ -0,0 +1,8 @@
+systemUnderAudit: nplstarter
+
+changesets:
+  - name: 1.0
+    changes:
+      - reset: true
+      - npl-load:
+        dir-list: npl
```

Comment:
Nit: when Github shows this little symbol in the diff, it means that there is no newline at the end of the file. Generally our convention is that all files should end with a newline (but it's not an important one).

![image](https://github.com/user-attachments/assets/12bfbf97-7b6d-4623-96b2-5fc2321c6f73)

Reply:
Fixed